Views/SubscribedEvents/Search/global.html.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+?>
+<?php if (!empty($showMore)): ?>
+    <a href="<?php echo $view['router']->generate('mautic_contactsource_index', ['search' => $searchString]); ?>"
+       data-toggle="ajax">
+        <span><?php echo $view['translator']->trans('mautic.core.search.more', ['%count%' => $remaining]); ?></span>
+    </a>
+<?php else: ?>
+    <a href="<?php echo $view['router']->generate(
+        'mautic_contactsource_action',
+        ['objectAction' => 'view', 'objectId' => $source->getId()]
+    ); ?>" data-toggle="ajax">
+        <span><?php echo $source->getName(); ?></span>
+        <?php
```

Comment:
Extra `<?php ?>` ?

Reply:
this has markup (html) interlaced with PHP, so there will be multiple <?php> tags, you can see that the previous php tag was closed (?>) right before the "</span>" on line 22.

Reply:
I get that, but that's an empty <?php ?> with nothing inside it though.... No <?php else: ?> or <?php echo "" ?>. Just a PHP tag with nothing inside

Reply:
An empty php tag just to make sure that PHP feels included.